Additional corrections go g (#26)
dot-layer/blog: content/blog/2018-09-01-grammar-of-graphics.md

<div id="fig:facet1" class="figure" style="text-align: center">
<img src="2018-09-01-grammar-of-graphics_files/figure-html/facet1-1.png" alt="It's difficult to see the true pattern in each group here"  />
</div>

> It's difficult to see the true pattern in each group here while it's quite clear it in the [figure below](#fig:facet2).

Reviewer: Je catch pas pourquoi cette phrase là est répété  tho. On la dit déjà avant le graphique. Moi je laisserais seulement:
```suggestion
while they are quite clear in the [figure](#fig:facet2) below.
```
